lib/idx/remediators/SelectEnrollmentChannel.ts

```diff
@@ -25,7 +26,18 @@ export class SelectEnrollmentChannel extends Remediator<SelectEnrollmentChannelV
   static remediationName = 'select-enrollment-channel';
 
   canRemediate() {
-    return Boolean(this.values.channel);
+    if (this.values.channel) {
+      return true;
+    }
+
+    if (this.values.authenticator) {
+      const { id, channel } = this.values.authenticator as Authenticator;
+      if (!!id && !!channel) {
```

Comment:
`||` instead of `&&`?

Reply:
both the `channel` and `id` are required

Reply:
nvm, I miss read `!!` to `!` ...